13_adult.do

```diff
@@ -46,19 +46,6 @@
 	replace a_bp_treat=1 if sh224==1 | sh324==1
 	}
 
-*a_bp_sys & a_bp_dial: 18y+ systolic & diastolic blood pressure (mmHg) in adult population 
```

Comment:
Could you clarify the intention of the modification here? @yining-sun

Reply:
I didn't intend to delete this line. I will fix it.